docs/resources/Webhook.md

```diff
@@ -104,7 +104,7 @@ Used to represent a webhook.
 Creates a new webhook and returns a [webhook](#DOCS_RESOURCES_WEBHOOK/webhook-object) object on success. Requires the `MANAGE_WEBHOOKS` permission. Fires a [Webhooks Update](#DOCS_TOPICS_GATEWAY_EVENTS/webhooks-update) Gateway event.
 
 An error will be returned if a webhook name (`name`) is not valid. A webhook name is valid if:
-- It does not contain the substring '**clyde**' (case-insensitive)
+- It does not contain the substrings "**clyde**" or "**discord**" (case-insensitive)
```

Comment:
```suggestion
- It does not contain the substrings `clyde` or `discord` (case-insensitive)
```
fine with this change—i know you're following the existing format, but i want to update it to be consistent with https://discord.com/developers/docs/resources/user#usernames-and-nicknames (so just changed to `)